_providers/nauta.cu.md

```diff
@@ -15,6 +15,13 @@ server:
 after_login_hint: |
     Atención - con nauta.cu, puede enviar mensajes sólo a un máximo de 20 personas a la vez.
     En grupos más grandes, no puede enviar mensajes o abandonar el grupo.
+config_defaults:
+  bcc_self: 0
+  sentbox_watch: 0
+  mvbox_watch: 0
+  mvbox_move: 0
```

Comment:
we have to options here:
1. enable `mvbox_move`
2. enable autodeletion at once

I would go for 2(remember nauta has only 100MB and users get full inbox and don't know how to fix (non-tech people)) but by your comments you don't want this for now so at least 1 will be needed, reason:
if messages stay in the inbox they will be re-downloaded by other email clients, so they need to be deleted or moved to DeltaChat folder

Reply:
well, it is just a general precaution not to enable auto-delete for everyone now.

i cannot really value the situation with nauta, how well it is already tested etc. seems to work at least for you :) is it already tested with some wider audience?

so, i would leave the decision for 1 or 2 up to you :)

Reply:
added a pr after some discussions on irc :)